_pydevd_bundle/pydevd_xml.py

```diff
@@ -128,32 +193,7 @@ def get_type(o):
         All container objects should have a resolver.
     """
 
-    try:
-        type_object = type(o)
-        type_name = type_object.__name__
-    except:
-        #This happens for org.python.core.InitModule
-        return 'Unable to get Type', 'Unable to get Type', None
-
-    try:
-
-        if type_name == 'org.python.core.PyJavaInstance':
-            return (type_object, type_name, pydevd_resolver.instanceResolver)
-
-        if type_name == 'org.python.core.PyArray':
-            return (type_object, type_name, pydevd_resolver.jyArrayResolver)
-
-        if _TYPE_MAP is None:
-            _update_type_map()
-        for t in _TYPE_MAP:
-            if isinstance(o, t[0]):
-                return (type_object, type_name, t[1])
-    except:
-        traceback.print_exc()
-
-    #no match return default
-    return (type_object, type_name, pydevd_resolver.defaultResolver)
-
+    return TypeResolveHandler.instance().get_type(o)
```

Comment:
`TypeResolveHandler.instance().get_type(o)` was used to avoid initializing `TypeResolveHandler` eagerly for two reasons

1) The old implementation did it lazily so I wanted to preserve that
2) I wanted to load the plugins only after the module is loaded to avoid potential problems with circular dependencies